agent.yaml

```diff
@@ -19,6 +19,18 @@ metrics:
             password: ${SUPABASESERVICEROLESECRET}
           static_configs:
             - targets: [${SUPABASEPROJECTURL}]
+        - job_name: supabase-awyuaxlpezyhfkpznrox-rr-eu-central-1-tuoer-scraper
+          scrape_timeout: 30s
+          scrape_interval: 30s
+          scheme: https
+          metrics_path: "/customer/v1/privileged/metrics"
+          basic_auth:
+            username: service_role
+            password: ${SUPABASESERVICEROLESECRET}
+          static_configs:
+            - targets: ["awyuaxlpezyhfkpznrox-rr-eu-central-1-tuoer.supabase.co"]
+              labels:
+                - supabase_project_ref: "awyuaxlpezyhfkpznrox-rr-eu-central-1-tuoer"
```

Comment:
Can we use variables e.g. `${SUPABASEPROJECTURL}` for some of these? Otherwise lgtm

Reply:
I agree it'd be better but I don't have access to the repo secrets and this is not a secret anyway.
Any chance you have admin access on this one ?

Reply:
I do not have access, but someone else does and is an easy add

Looks like they are part of the `infra` repo though:
- https://github.com/WalletConnect/infra/blob/3c2d7dab2d1db1c20eb7f8d4a5fd67ae8ecd6153/terraform/supabase_metrics/main.tf#L47
- https://github.com/WalletConnect/infra/blob/3c2d7dab2d1db1c20eb7f8d4a5fd67ae8ecd6153/.github/workflows/cd.yml#L13

If not secret then not a big deal I suppose.